code/datums/diseases/advance/symptoms/symptoms.dm

```diff
@@ -60,6 +60,11 @@
 		next_activation = world.time + rand(symptom_delay_min * 10, symptom_delay_max * 10)
 		return TRUE
 
+/datum/symptom/proc/on_stage_change(new_stage, datum/disease/advance/A)
+	if(neutered)
+		return FALSE
+	return TRUE

```

Comment:
This return value isn't used?

Reply:
isnt it used on oxygen.dm line 55 to tell it not to add the trait if it is neutered?

Reply:
If you're giving the base proc return values you should do so for children overwrites.

Reply:
im confused, the code that is handled similair in symptoms use the base proc as a means of checking if it is neutered without having return values on their children shouldn't i implement it the same way or would giving the children return values improve the code in a way i am not seeing?

Reply:
It's bikeshed, I just like return values

Reply:
although i'm assuming it caused the issue here.

Reply:
ill add them then i just didnt know if it was like a dm thing that made it matter